web-ui/src/main/resources/catalog/views/default/templates/recordView/summary.html

```diff
@@ -44,21 +44,17 @@ <h3 data-translate="">tempExtent</h3>
     </span>
     <div>
       <h3 data-translate="">resourceVerticalRange</h3>
-      <div
-        class="flex flex-row flex-align-center gn-vertical-range"
-        data-ng-repeat="range in mdView.current.record.resourceVerticalRange"
-      >
-        <div class="width-25">
-          <i class="fa fa-fw fa-chevron-up" />
-        </div>
-        <div class="width-75">
-          <div data-ng-show="range.lte">
-            <dt>{{range.lte}}</dt>
-          </div>
-          <div data-ng-show="range.gte">
-            <dt>{{range.gte}}</dt>
-          </div>
-        </div>
+      <div class="flex flex-col">
+        <p data-ng-repeat="range in mdView.current.record.resourceVerticalRange">
+          <span data-ng-show="range.gte || range.gte===0"> {{range.gte}}m </span>
```

Comment:
For units, we can probably default it to meter but it may be something else, so maybe we should add that info during indexing of the value.

It is odd because in ISO19115 there is a unitOfMeasure that we don't have in the XSD
![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/03110720-2303-4773-9ded-889814423bb9)

And in ISO19115-3, CRS can provide the information about the unit
![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/c6860b6b-8e4d-4421-95c0-65794a25c2b8)

Reply:
Am I correct in saying GeoNetwork currently does not allow to define a unitOfMeasure in the editor for vertical extents?

Trying to figure out what elements of the schemas need to be picked for this purpose, but not sure where to look.

Reply:
I think when you choose a vertical CRS eg. https://epsg.io/9288 you've the unit defined. Probably in most cases it will be "m" so we can start with this as a default value (I don't have in mind any records with something else for now).

Reply:
Should we then introduce `unit`  in the index, with a hardcoded `m` value? Or shall we keep the change as it is, until the need arises to have other units?

Reply:
In the latest commit we (Mathieu and I) modified the frontend to fetch the unit from the index, with a fallback `m` value. Then we are ready for schemas that eventually want to / should have a tailored unit. The unit was added as an example in the -3 schema.